web-app/src/main/java/com/infoshare/webapp/service/QuestionService.java

```diff
@@ -21,31 +22,36 @@ public QuestionService(ReadFileService readFileService) throws IOException {
         this.readFileService = readFileService;
         questionsList = this.readFileService.loadQuestions();
     }
-    public List<Questions> getAll(){
+
+    public List<Questions> getAll() {
         return questionsList;
     }
+
     public void editQuestionById(Long id, Questions question) {
```

Comment:
```suggestion
    public void editQuestion(Long id, Questions question) {
```
wiadomo, że jest po Id inaczej się nie da

Reply:
Nazwy poprawione